src/App.tsx

```diff
@@ -1,13 +1,23 @@
-import Logo from "@/assets/logo.png";
-import HelloWorld from "@/components/HelloWorld/HelloWorld";
+import { useEffect, useState } from "react";
 
 import styles from "./App.module.css";
+import Login from "./components/Login/Login";
+import Products from "./components/Products/Products";
 
 export default function App() {
+  const [isLoggedin, setIsLoggedin] = useState<boolean>(false);
```

Comment:
What happens if the user has no valid credentials? Please include the error handling.